src/Language/Fixpoint/Types/Solutions.hs

```diff
@@ -132,27 +162,37 @@ instance PPrint QBind where
 -- | A `Sol` contains the various indices needed to compute a solution,
 --   in particular, to compute `lhsPred` for any given constraint.
 --------------------------------------------------------------------------------
-data Sol a = Sol
+data Sol b a = Sol
```

Comment:
Where is your remark? Yes, I do need it, because without Gradual Typing this is a `()` but with I use `b` to store the gradual solutions.